syntax = "proto3";

package room;

message Room {
    int32 room_id = 1;          // Oda ID'si
    string room_name = 2;       // Oda adı
    int32 max_users = 3;        // Maksimum kullanıcı sayısı
    bool is_private = 4;        // Gizli oda durumu
}
